packages/phoenix-event-display/src/tests/loaders/script-loader.test.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment jsdom
+ */
+import { ScriptLoader } from '../../loaders/script-loader';
+
+describe('ScriptLoader', () => {
+  let scriptLoader: ScriptLoader;
+
+  beforeEach(() => {
+    scriptLoader = new ScriptLoader();
+  });
+
+  it('should create an instance', () => {
+    expect(scriptLoader).toBeTruthy();
+  });
+
+  it('should load a script dynamically from a URL', () => {
+    const scriptURL = 'https://www.google.com/jsapi';
+    const scriptFor = 'google_jsapi';
+    const parentElement = document.getElementsByTagName('head')[0];
+
+    jest.spyOn(ScriptLoader, 'loadScript');
+    ScriptLoader.loadScript(scriptURL, scriptFor, parentElement);
+    expect(ScriptLoader.loadScript).toHaveBeenCalledWith(
```

Comment:
This also just asserts if the previous statement executed. If you call a function, it is going to be called, there is no need to assert that here.

What we need to assert here is that the script is correctly loaded in the DOM.